spartan/aztec-network/templates/_helpers.tpl

```diff
@@ -222,7 +222,7 @@ nodeSelector:
 {{- end -}}
 
 {{- define "aztec-network.waitForEthereum" -}}
-if [ -n "${EXTERNAL_ETHEREUM_HOSTS}" ]; then
```

Comment:
Adding the addresses script set -u in the shell, which made this check stricter, added a unset if unset